executable-spec/src/Cardano/Ledger/Spec/SM/Ideation.hs

```diff
@@ -1,20 +1,29 @@
+{-# LANGUAGE DataKinds #-}
 {-# LANGUAGE OverloadedStrings #-}
 
 module Cardano.Ledger.Spec.SM.Ideation where
 
 import           Control.State.DataAutomaton
+import           Control.State.DataAutomaton.Interpreter.Run
+
 
 sipCommitReveal :: DataAutomaton
 sipCommitReveal =
   DataAutomaton
   { start = "Idle"
   , transitions =
-    [ Transition
+    [
+      Transition
       { from = "Idle"
       , guard = Const True
       , act = Indexed "Tick" "s"
-      , update = Do ("cslot" := (VarE "s"))
+      , update = ["cslot" := (VarE "s" :+ Const (10 :: Int))]
```

Comment:
Why do we need to add the constant 10 to the current slot?

Reply:
These are just examples at the moment. They aren't the actual models for our update mechanism.